### PR TITLE
Feature/leaderboard rewards part2

### PR DESCRIPTION
**Pull request by ibrahimnadra**
1. Reorganised the folder structure.
2. Renamed the folder.
3. Modified the fe test cases and page objects.

### CODE REVIEW

File: go1percent/test/Go1PercentFEAutomation/Leaderboard_rewards_part2/AddRewardTest.js
Comment (Ritika-gupta20) [inline go1percent/test/Go1PercentFEAutomation/Leaderboard_rewards_part2/AddRewardTest.js:1]: Please add these test in one folder names as Leaderboard_rewards, under that the test file may contain 1 or 2 in the name to differentiate

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Please do the changes as per comments

File: go1percent/test/Go1PercentAPIAutomation/Leaderboard_rewards_part2/RedeemRewardTest.js
[the remark this answers is not in the document]
Comment (ibrahimnadra) [reply go1percent/test/Go1PercentAPIAutomation/Leaderboard_rewards_part2/RedeemRewardTest.js:1]: Done

File: go1percent/helpers/Go1PercentFEAutomation/TECHHUB/AdminUser/go1PercentTechhub.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 module.exports = {
 
-
```
Comment (Ritika-gupta20) [inline go1percent/helpers/Go1PercentFEAutomation/TECHHUB/AdminUser/go1PercentTechhub.js:3]: Why any change is done here in other module

**Review by Ritika-gupta20 — APPROVED** [review]

File: go1percent/helpers/Go1PercentFEAutomation/LeaderboardRewards/helperFunctions.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  generateRandomString: function () {
+    const allowedChars = 'abcdefghijklmnopqrstuvwxyz';
+    let randomString = '';
+  
+    // Generate the first character separately to ensure it's capitalized
+    randomString += allowedChars.charAt(Math.floor(Math.random() * allowedChars.length)).toUpperCase();
+  
+    // Generate the rest of the string
+    for (let i = 1; i < 6; i++) {
```
Comment (deepansh257) [inline go1percent/helpers/Go1PercentFEAutomation/LeaderboardRewards/helperFunctions.js:11]: Do not use variable names like i, j, k. Instead give meaningful names

File: go1percent/test/Go1PercentAPIAutomation/LeaderboardRewards/RedeemRewardTest.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  //PUT the redeemed rewards (removes or mark them processed)
+  // it('Should update the redeemed reward to processed', async function ({ supertest }) {
+  //   const startTime = Date.now();
+  //   await supertest
+  //       .request(globals.Add_Contribution.BaseUrl)
+  //       .put("/updateRedeemReward")
+  //       .set(header)
+  //       .send({"rewardId":324,"rewardType":"Individual","status":"Processed","redeemId":219})
+  //       .expect(200)
+  //       .then(function (response) {
```
Comment (deepansh257) [inline go1percent/test/Go1PercentAPIAutomation/LeaderboardRewards/RedeemRewardTest.js:192]: Remove this code if not needed. Else uncomment it if useful

**Review by deepansh257 — CHANGES_REQUESTED** [review]

**Review by Sparsh79 — APPROVED** [review]

**Pull request merged**